sway/input/cursor.c

```diff
@@ -473,7 +473,7 @@ static void handle_touch_down(struct wl_listener *listener, void *data) {
 	seat->touch_x = lx;
 	seat->touch_y = ly;
 
-	if (surface && wlr_surface_accepts_touch(wlr_seat, surface)) {
+	if (surface && wlr_surface_accepts_touch(wlr_seat, surface) && false) {
```

Comment:
I think this slipped in by accident?

Reply:
Oops, good catch! It's a remnant from my testing session. Removed.